src/ja.json

```diff
@@ -0,0 +1,217 @@
+{
+      "welcome-to-rainway" : "Rainwayへようこそ",
+      "no-account-yet" : "まだアカウントを持っていない?",
+      "register-now" : "今すぐ登録",
+      "forgot-password" : "パスワードを忘れた",
+      "register-new-account" : "新しくアカウントを登録",
+      "reset-your-password" : "パスワードをリセットする",
+      "email" : "メールアドレス",
+      "password" : "パスワード",
+      "username" : "ユーザーネーム",
+      "confirm-password" : "パスワードを確認する",
+      "sign-in" : "サインイン",
+      "register" : "登録",
+      "reset-password" : "パスワードリセット ",
+      "save-changes" : "変更を保存",
+      "cancel" : "キャンセル",
+      "submit" : "送信",
+      "download-dashboard" : "ダッシュボードをダウンロード",
+      "support" : "サポート",
+      "finding-servers" : "サーバーを探しています...",
+      "webrtc-connection-failed" : "WebRTCとの接続に失敗しました",
+      "lost-webrtc-connection" : "WebRTCとの接続が失われました.",
+      "connecting" : "接続しています",
+      "no-online-dashboard" : "オンラインになっているダッシュボードが見つかりません",
+      "no-dashboard-message" : "Rainwayクライアントでゲームをストリーミングしてプレイするには、ゲームがインストールされているWindows10を搭載したコンピュータにRainwayダッシュボードをインストール必要があります。これは1度だけのセットアップで、完了には1分もかかりません。",
+      "log-out" : "ログアウト",
+      "or" : "または",
+      "go-back" : "戻る",
+      "outdated-server" : "サーバーのバージョンが古くなっています!",
+      "outdated-server-message" : "このホストで実行されているサーバーのバージョンとは互換性がない原因となった、Rainwayクライアントの大きな変更があります。このコンピュータに接続する前に、Rainwayサーバーを更新してください。",
+      "server-running-here" : "このコンピュータでは既にサーバーが稼働しています!",
+      "server-running-here-message" : "このサーバーをホストしているコンピュータでRaiwayクライアントを使用しようとしています。Rainwayダッシュボードを使用すると、このコンピュータでゲームをローカルに起動できます。Rainwayクライアントでゲームをストリーミングするには、サーバーをホストしているコンピュータ以外のデバイスから接続してください。",
+      "all-games" : "すべてのゲーム",
+      "we-found-games-message" : "システムに{{count}}のゲームがインストールされています",
+      "we-found-games-message_plural" : "システムに{{count}}のゲームがインストールされています",
```

Comment:
It's OK to delete this line, and others like it: i18n knows to never check the `_plural` string in a Japanese locale.